Author: Andreas Tille <tille@debian.org>
        Aaron M. Ucko <ucko@debian.org>
Last-Update: 2023-01-15 20:56:15
Description: Make sure javadoc builds properly

Index: b/ngs/ngs-java/CMakeLists.txt
===================================================================
--- a/ngs/ngs-java/CMakeLists.txt
+++ b/ngs/ngs-java/CMakeLists.txt
@@ -140,11 +140,11 @@ if ( Java_FOUND )
             set(SRC_FILES "${SRC_FILES} ${f}")
         endforeach()
 
-        if( ${OS} STREQUAL "windows" )
+        # if( ${OS} STREQUAL "windows" )
             set( NGS_JAVADOC_DIR ${TARGDIR} )
-        else()
-            set( NGS_JAVADOC_DIR ${TARGDIR}/obj )
-        endif()
+        # else()
+        #     set( NGS_JAVADOC_DIR ${TARGDIR}/obj )
+        # endif()
         if ( Java_JAR_EXECUTABLE AND NOT (${OS} STREQUAL "windows") )
             add_custom_target(
                 ngs-doc-jar ALL COMMAND
